packaging/conda/bld.bat

```diff
@@ -2,7 +2,7 @@ REM Echo all output
 @echo on
 
 REM Build the Khiops binaries
-cmake --preset windows-msvc-release -DBUILD_JARS=OFF -DTESTING=OFF
+cmake --preset windows-msvc-release -DBUILD_JARS=OFF -DTESTING=OFF -A "" -T ""
```

Comment:
I'd leave a comment on why we need these options.

Reply:
OK, I'd copy the commit message as a comment here.

Reply:
Done.